sdk/search/azure-search-documents/README.md

```diff
@@ -206,11 +194,9 @@ for (SearchResult searchResult: searchResultsIterable) {
 
 ## Key concepts
 
-An Azure Cognitive Search service contains one or more indexes that provide
-persistent storage of searchable data in the form of JSON documents.  _(If
-you're brand new to search, you can make a very rough analogy between
-indexes and database tables.)_  The Azure.Search.Documents client library
-exposes operations on these resources through two main client types.
+An Azure Cognitive Search service contains one or more indexes that provide persistent storage of searchable data in 
+the form of JSON documents. _(If you're new to search, you can make a very rough analogy between indexes and database 
+tables.)_ The Azure.Search.Documents client library exposes operations on these resources through two main client types.
```

Comment:
Azure.Search.Documents? It looks .NET ish.

Reply:
I'll take a good bet that this was copied from .NET.